Easydict/Feature/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -694,9 +694,14 @@ - (void)selectTextTranslate {
     NSLog(@"selectTextTranslate windowType: %@", @(windowType));
     self.eventMonitor.actionType = EZActionTypeShortcutQuery;
     [self.eventMonitor getSelectedText:^(NSString *_Nullable text) {
-        // If text is nil, currently, we choose to clear input.
-        self.selectedText = [text trim] ?: @"";
         self.actionType = self.eventMonitor.actionType;
+        
+        // Clear query if text is nil and user don't want to keep the last result.
+        if (!text && !Configuration.shared.keepPrevResultWhenEmpty) {
+            text = @"";
+        }
+        self.selectedText = [text trim];
+        
```

Comment:
@yam-liu For [suggested changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes), you can just commit changes without submitting another one.

Reply:
Got it, but I love rebase more and keeping the commit history tidy. 😂

Reply:
No, we shouldn't use `git rebase` again for a branch that has already been pushed to a remote branch, unless you're the only one using it https://github.com/tisfeng/Easydict/pull/240#issuecomment-1837151779

In addition, making code changes after each review become a commit, which is good for other collaborators' code review instead of a blob of code.

Reply:
I partially agree with you.